src/pages/application-inventory/application-inventory.tsx

```diff
@@ -410,9 +410,11 @@ export const ApplicationInventory: React.FC = () => {
             }
             noDataState={
               <NoDataEmptyState
+                // t('terms.applications')
```

Comment:
worth removing the comment?

Reply:
@mrizzi i18next-parser is the tool used for detecting which are the new "internationalization" keys and add them automatically to the .json files.

In this specific case the line 415 `what: t("terms.applications").toLowerCase(),` is not inside a React component neither is referenced directly in a JS file but it is part of a JSON parameter passed in line 414; as a result of this, the command `yarn extract` won't be able to detect `terms.applications`, adding a comment helps the tool to detect the key and add it to the .json internationalization files. For more info about this see https://github.com/i18next/i18next-parser#caveats

This doesn't mean that we need to add comments everywhere; whenever the user creates a commit the `husky` config will edit the files just like the video I shared in this PR and let the user know which are keys detected in your PR. So these 2 comments in this PR are needed to persist the key `terms.application`

Reply:
awesome, thanks @carlosthe19916 :+1: